src/unix/core.c

```diff
@@ -198,6 +198,19 @@ void uv__make_close_pending(uv_handle_t* handle) {
   handle->loop->closing_handles = handle;
 }
 
+int uv__getiovmax(void) {
+#if defined(IOV_MAX)
+  return IOV_MAX;
+#elif defined(_SC_IOV_MAX)
+  static int iovmax = -1;
+  if (iovmax == -1)
+    iovmax = sysconf(_SC_IOV_MAX);
```

Comment:
`The sysconf() function provides a method for applications to determine the current value of a configurable system limit or option variable.`

shouldn't we be passing this value in as an option and configuring the max, as a user of lubuv?

some systems, like osx are notorious for defaulting to low system configuration values.